src/pybind/mgr/dashboard/controllers/daemon.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+
+from ..exceptions import DashboardException
+from ..security import Scope
+from ..services.exception import handle_orchestrator_error
+from ..services.orchestrator import OrchClient, OrchFeature
+from . import APIDoc, APIRouter, RESTController
+from ._version import APIVersion
+from .orchestrator import raise_if_no_orchestrator
+
+
+@APIRouter('/daemon', Scope.HOSTS)
+@APIDoc("Perform actions on daemons", "Daemon")
+class Daemon(RESTController):
+    @raise_if_no_orchestrator([OrchFeature.DAEMON_ACTION])
+    @handle_orchestrator_error('daemon')
+    @RESTController.MethodMap(version=APIVersion.EXPERIMENTAL)
```

Comment:
Is experimental the new black? :P Now serious: I see the point of always starting with experimental version, but then we'll have to force ourselves to periodically review and freeze endpoints as stable.